stock_prices/stock_prices.py

```diff
@@ -3,13 +3,26 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
-
+# need a high an a low for a sale
+# buy before sell
+# find the max difference = high - low
+	max = None
```

Comment:
You're on the right track for this function. 
What is the goal of setting a variable to `None`??